----------------------------------------------------------------
Tue Jul 26 14:06:14 BRT 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a816c00e-0182-3b79-2e14-0000009964e8 
on database directory C:\Users\deocl\Documents\SI\2022-1\dcc193-webII\Atv6\dcc193-2022-1-atv6-DeoclecioFilho\tomato\target\database.db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@6bc168e5 
Loaded from file:/C:/Users/deocl/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Eclipse Adoptium
java.runtime.version=17.0.3+7
user.dir=C:\Users\deocl\Documents\SI\2022-1\dcc193-webII\Atv6\dcc193-2022-1-atv6-DeoclecioFilho\tomato
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Tue Jul 26 14:31:02 BRT 2022:
Shutting down instance a816c00e-0182-3b79-2e14-0000009964e8 on database directory C:\Users\deocl\Documents\SI\2022-1\dcc193-webII\Atv6\dcc193-2022-1-atv6-DeoclecioFilho\tomato\target\database.db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@6bc168e5 
Tue Jul 26 14:31:02 BRT 2022 Thread[Thread-19,5,main] Cleanup action starting
ERROR XBM0I: Directory target/database.db cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Cleanup action completed
Tue Jul 26 14:31:02 BRT 2022 Thread[Thread-19,5,main] Cleanup action starting
ERROR XBM0I: Directory target/database.db cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Cleanup action completed
----------------------------------------------------------------
Tue Jul 26 14:31:11 BRT 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 3a7f0048-0182-3b79-2e14-0000009964e8 
on database directory C:\Users\deocl\Documents\SI\2022-1\dcc193-webII\Atv6\dcc193-2022-1-atv6-DeoclecioFilho\tomato\target\database.db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@6bc168e5 
Loaded from file:/C:/Users/deocl/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Eclipse Adoptium
java.runtime.version=17.0.3+7
user.dir=C:\Users\deocl\Documents\SI\2022-1\dcc193-webII\Atv6\dcc193-2022-1-atv6-DeoclecioFilho\tomato
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Tue Jul 26 14:50:09 BRT 2022:
Shutting down instance 3a7f0048-0182-3b79-2e14-0000009964e8 on database directory C:\Users\deocl\Documents\SI\2022-1\dcc193-webII\Atv6\dcc193-2022-1-atv6-DeoclecioFilho\tomato\target\database.db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@6bc168e5 
Tue Jul 26 14:50:09 BRT 2022 Thread[Thread-34,5,main] Cleanup action starting
ERROR XBM0I: Directory target/database.db cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Cleanup action completed
Tue Jul 26 14:50:09 BRT 2022 Thread[Thread-34,5,main] Cleanup action starting
ERROR XBM0I: Directory target/database.db cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Cleanup action completed
----------------------------------------------------------------
Tue Jul 26 14:50:15 BRT 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 1c16805c-0182-3b79-2e14-0000009964e8 
on database directory C:\Users\deocl\Documents\SI\2022-1\dcc193-webII\Atv6\dcc193-2022-1-atv6-DeoclecioFilho\tomato\target\database.db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@6bc168e5 
Loaded from file:/C:/Users/deocl/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Eclipse Adoptium
java.runtime.version=17.0.3+7
user.dir=C:\Users\deocl\Documents\SI\2022-1\dcc193-webII\Atv6\dcc193-2022-1-atv6-DeoclecioFilho\tomato
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
